hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileUpdateHandler.java

```diff
@@ -49,7 +89,105 @@ public void preHandle(@NonNull final PreHandleContext context) {
      *
      * @throws NullPointerException if one of the arguments is {@code null}
      */
-    public void handle() {
-        throw new UnsupportedOperationException("Not implemented");
+    public void handle(
+            @NonNull final HandleContext handleContext,
+            @NonNull final FileUpdateTransactionBody fileUpdate,
+            @NonNull final WritableFileStoreImpl fileStore,
+            @NonNull final FileServiceConfig fileServiceConfig) {
```

Comment:
by merging develop in your branch you should be able to get the FileServiceConfig out of the HandleContext

Reply:
done and used it